code/game/gamemodes/miniantags/slaughter/slaughterevent.dm

```diff
@@ -2,9 +2,9 @@
 /datum/round_event_control/slaughter
 	name = "Spawn Slaughter Demon"
 	typepath = /datum/round_event/slaughter
-	weight = 0 //Admin only
+	weight = 1 //Very rare
 	max_occurrences = 1
-	earliest_start = 30 //Spawn later with more blood, hopefully
+	earliest_start = 0
```

Comment:
This is not working aswell, all slaughter demon random events happened BEFORE 30 minutes, in like 10 minutes into the round